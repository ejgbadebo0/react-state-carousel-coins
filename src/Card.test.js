import React from "react";
import { render } from "@testing-library/react";
import image1 from "./image1.jpg";
import Card from './Card';

it('renders', () => {
    render(<Card src={image1} caption='Caption for rendering.'/>);
    /*
    const { getByText } = render(<Card src={image1} caption='Caption for rendering.'/>);
    const comp = getByText('Card');
    expect(comp).toHaveClass('Card');
    expect(comp).toBeInDocument();*/
})

it('matches snapshot', () => {
    const { asFragment } = render(<Card src={image1} caption='Caption for rendering.'/>);
    expect(asFragment()).toMatchSnapshot();
})