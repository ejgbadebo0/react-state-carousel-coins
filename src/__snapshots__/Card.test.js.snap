// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Caption for rendering.
    </h4>
    <img
      alt="Caption for rendering."
      class="Card-image"
      src="image1.jpg"
    />
    <small
      class="Card-small"
    >
      Image  of .
    </small>
  </div>
</DocumentFragment>
`;
